_includes/cloud-org/cloud-payment-details.md

```diff
@@ -0,0 +1,2 @@
+* FeatureBase supports all major credit cards
+* Only US bank accounts are accepted at this time
```

Comment:
If this is the case, then it needs to be added to the featurebase payments page:

https://www.featurebase.com/pricing